### PR TITLE
fix(*): resolve issues of india release 

### PR DESCRIPTION
**Pull request by sachinrameshn**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master` 
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | MANAGER-12388, MANAGER-12389, MANAGER-12394, MANAGER-12393 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description
resolve issues after manager india integration

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/add/dedicatedCloud-datacenter-add.controller.js [hunk 1]
```diff
@@ -96,7 +97,7 @@ export default class {
 
   convertPriceValueToDisplay(value) {
     const priceAsString = new Intl.NumberFormat(
-      this.ovhSubsidiary.toLowerCase(),
+      this.coreConfig.getUserLocale().replace('_', '-'),
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/add/dedicatedCloud-datacenter-add.controller.js:100]: **nitpick:** The price should always be based on Subsidiary. If a French user, switches to Italian language. the price displayed will not be consistent.

**Review by anooparveti — CHANGES_REQUESTED** [review]

**Review by acavalloni — APPROVED** [review]

File: packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/add/dedicatedCloud-datacenter-add.controller.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (sachinrameshn) [reply packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/add/dedicatedCloud-datacenter-add.controller.js]: Done Anoop.

**Review by anooparveti — APPROVED** [review]

**Review by darsene — DISMISSED** [review]

File: packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/dedicatedCloud-datacenter.constants.js [hunk 2]
```diff
@@ -28,9 +28,12 @@ export const TRACKING_NEW_PRODUCT_BANNER = {
     'dedicated::dedicatedCloud::details::datacenter::link-to-vmware-products',
 };
 
+export const OVERRIDE_IN = { IN: `en-IN` };
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/dedicatedCloud-datacenter.constants.js:31]: In future we might need to add override for other countries as well. So dont use country specific variables.
```suggestion
export const LANGUAGE_OVERRIDE = { IN: `en-IN` };
```

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/dedicatedCloud-datacenter.constants.js (hunk 2 above)
Comment (sachinrameshn) [reply packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/dedicatedCloud-datacenter.constants.js]: Done Anoop

**Review by anooparveti — APPROVED** [review]

**Pull request merged**